src/Symfony/Component/Validator/Constraints/Range.php

```diff
@@ -57,6 +58,7 @@ class Range extends Constraint
      * @param string|null              $maxPropertyPath        Property path to the max value
      * @param string[]|null            $groups
      */
+    #[HasNamedArguments]
     public function __construct(
         ?array $options = null,
```

Comment:
We do not have a way to deprecate certain arguments only in Symfony with annotations/attributes, do we?
We could use https://blog.jetbrains.com/phpstorm/2020/10/phpstorm-2020-3-eap-4/#deprecated for this in theory.

Reply:
In theory that's possible by changing the order of the arguments, remove the type-hints and then determine the order in which arguments have been passed by inspecting the actual types. But I am afraid we won't be able to achieve that in the constraints due to the payload always accepting every type.